commons-packet/commons-packet-service/pom.xml

```diff
@@ -269,4 +269,80 @@
                   <organizationUrl>https://github.com/mosip/commons</organizationUrl>
                 </developer>
         </developers>
+    <profiles>

```

Comment:
auth adapter is a runtime dependency can not be added here, plus version can not be 1.1.4.